crates/symbolicator/src/services/cacher.rs

```diff
@@ -186,9 +186,11 @@ impl<T: CacheItemRequest> Cacher<T> {
         }
 
         // This is also reported for "negative cache hits": When we cached the 404 response from a
-        // server as empty file.
+        // server as empty file, as well as other download errors.
         metric!(counter(&format!("caches.{}.file.hit", name)) += 1);
         metric!(
+            // TODO: what does it mean when this is a cache status that wraps around variable length
+            // content? should we fix these to just the marker length if this is Malformed and DownloadError?
```

Comment:
i don't understand this question

Reply:
i may have misread this and thought `name` was the cache status for some reason, or this comment was placed in the wrong place and might belong in `check_expiry` and made its way here. either way, it's no longer relevant so i'll just delete it 🙃 sorry about the confusion